charts/openyurt/templates/yurt-manager-auto-generated.yaml

```diff
@@ -824,7 +1029,7 @@ webhooks:
   name: validate.core.v1.pod.openyurt.io
   rules:
   - apiGroups:
-    - ""
+    - core.openyurt.io
```

Comment:
the group of pod is "" instead of `core.openyurt.io`

Reply:
done